setup.py

```diff
@@ -2,20 +2,24 @@
 # -*- encoding: utf-8 -*-
 from __future__ import absolute_import, print_function
 
+import os
+
 from setuptools import find_packages, setup
 
 setup(
     name='watchmaker',
-    version='0.0.1',
+    version='0.1',
     author='Plus3IT Maintainers of Watchmaker',
     author_email='projects@plus3it.com',
     url='https://github.com/plus3it/watchmaker',
     packages=find_packages('src'),
     package_dir={'': 'src'},
     include_package_data=True,
+    scripts=[os.path.join('scripts', 'Watchmaker.py')],
```

Comment:
Surely there is but I would like to resolve the creation with this, and the open a separate issue for the work around to get the PR in.  Ill do some comparison with the entrypoint option.  Ill create a smaller issue for it.